packages/web-app-files/src/composables/useSort/index.ts

```diff
@@ -0,0 +1,117 @@
+import { ref, computed, ComputedRef, unref, Ref, isRef } from '@vue/composition-api'
+import { MaybeRef, MaybeReadonlyRef } from '../utils'
+
+import orderBy from 'lodash-es/orderBy'
+
+export enum SortDir {
+  Desc = 'desc',
+  Asc = 'asc'
+}
+
+interface SortField {
+  name: MaybeRef<string>
+  sortable?: MaybeRef<boolean>
```

Comment:
`sortable` can also be a callback which is intended to transform a value into a sortable value.